src/ranking_collecter/ranking_collector.py

```diff
@@ -0,0 +1,26 @@
+from string import Template
+import requests
+import user_info
+
+def collect_ranks(event, context):
+    usernames = user_info.USERNAMES
+    weekly_rankings = []
+    contest = user_info.CONTEST
+    weeklyContest = user_info.WEEKLYCONTEST
+
+    temp = Template('https://raw.githubusercontent.com/AWice/leetcode/master/leaderboard/scraped/${contestNo}weekly-contest-${weeklyContestNo}.txt')
+    url = temp.substitute(contestNo=contest, weeklyContestNo=weeklyContest)
```

Comment:
should be using string interpolation here; see [this article](https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36)